### ISSUE TITLE
Help in the Getting Started

### ISSUE THREAD

**Issue by angeliski** [viljamis/vue-design-system#96]
Hey @viljamis Really great job. 
The _vue-design-system_ is amazing!
I would like some help in the getting started... 
I have in my stack a project who have my components (e.g: **erp-components**) and I would like to create the design system using that lib. 

How I can do that? 

I need put my project **erp-components** inside the _vue-design-system_ (I would like to avoid that) or I can create a new project using the _vue-design-system_ and import the components... I Read the Getting Started but not is clearly to me.
You can provide some example external the _vue-design-system_

Anyway, thanks for share the project.

**Comment by viljamis** [comment]
@angeliski Hey! Just returned from a 5 week “offline” trip, hence the little delay here. Let me get back to this early next week when I’ve survived the jet lag :)

**Comment by angeliski** [comment]
Hey @viljamis Any update?

**Comment by viljamis** [comment]
@angeliski So sorry for the long delay! First question: is “erp-components” a vue project?

**Comment by angeliski** [comment]
hey @viljamis Do you have any ideia or feedback?

**Comment by viljamis** [comment]
@angeliski Just to make sure I understand correctly, you want to import components from vue design system into the erp-components project?

**Comment by angeliski** [comment]
I want to import erp-components into Vue Design System to create a documentation :)

**Comment by viljamis** [comment]
@angeliski If you only want to create a documentation I think it would make more sense to install `vue-styleguidist` to your erp-components project via npm and let that handle it for you?

**Comment by angeliski** [comment]
Yeah, I can run the `vue-styleguidist`, but I like to use `vue-design-system`. I prefer vue-design-system because he has more semantic for design team, and I don't need configure a lot of things.
I don't can run `vue-design-system` with custom components, from another package?

**Comment by angeliski** [comment]
Thank you @viljamis I take a look
